externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/operation/SparkOperationSuite.scala

```diff
@@ -76,6 +77,10 @@ class SparkOperationSuite extends WithSparkSQLEngine with HiveMetadataTests with
       .add("c16", "binary", nullable = false, "16")
       .add("c17", "struct<X: string>", nullable = true, "17")
 
+    if (SPARK_ENGINE_MAJOR_MINOR_VERSION._1 >= 3 && SPARK_ENGINE_MAJOR_MINOR_VERSION._2 > 2) {
```

Comment:
this does not work for 4.x, shall be (x>3 || (x==3 && y >=3)

Reply:
> 

ok

Reply:
Isn't is this : (major == 3 && minor >= 3)

Reply:
oops，I get you